www/install/php/Update-20.04.0-beta.1.php

```diff
@@ -30,3 +30,46 @@
         "UPGRADE : 19.10.2 Unable to set default contact_autologin_key"
     );
 }
+
+
+// Move broker xml files to json format
```

Comment:
why not add try/catch clauses and failing log message. This help to debug when something wrong occurs

Reply:
because IMO it's not especially a good approach
Because if update failed, it looks good on UI, and nobody will take care about issues during upgrade ;)

Reply:
but if it fails in between the two updates, won't the result be worse ?